### DISCUSSION TITLE
Prevent major updates on packageRule?

### DISCUSSION THREAD

**Discussion by bruderstein** [renovatebot/renovate#10045] [Request Help] [answered]
Is there a way to prevent major updates on a group of dependencies matched in a `packageRule`?

For example, I have dependencies:

```
"webpack": "4.42.0",
"style-loader": "1.22.0",
"postcss-loader": "3.0.0",
"foo-loader": "7.0.0"
```

And I'd like to write a package rule a bit like this:
```
  "packageRules": [
     {
       "matchPackagePatterns": ["^webpack$", "-loader$"],
       "something_that_says_what_types_to_update": ["patch", "minor"]
      }
  ]
```

Is the only way to limit these dependencies individually with `allowedVersions`? Or does something like this exist?

Thanks

Dave

**Comment by rarkins** [comment, marked as answer]
Renovate's default behavior is to update things. To stop updating things, you need to disable things in a rule, like this:

```
  "packageRules": [
     {
       "matchPackagePatterns": ["^webpack$", "-loader$"],
       "updateTypes": ["major"],
       "enabled": false
      }
  ]

```

**Reply by bruderstein** [reply]
Perfect, thank you for the super quick answer!

**Comment by HonkingGoose** [comment]
I was thinking the "Renovate dashboard approval" workflow might be a better fit?

```json
  "packageRules": [
    {
      "matchPackagePatterns": ["^webpack$", "-loader$"],
      "updateTypes": ["major"],
      "dependencyDashboardApproval": true
    }
  ]
```

Now major updates for those packages require dashboard approval. This means that you can tick a checkbox on the Renovate dashboard in case you ever **do want** to upgrade those dependencies.

As an added bonus: you now have visibility into new major versions for those packages via the dashboard, instead of just outright "banning" updates.
